lib/msal-browser/test/cache/TokenCache.spec.ts

```diff
@@ -56,6 +56,7 @@ describe("TokenCache tests", () => {
             storeAuthStateInCookie: false,
             secureCookies: false,
             cacheMigrationEnabled: false,
+            claimsBasedCachingEnabled: false,
```

Comment:
Any new tests to validate the bug you're fixing here?

Reply:
There's a few on `SilentFlowClient.spec.ts`, sorry for the massive diff, the automatic format on save has been doing that to all my PRs lately.

Reply:
They do cover the following cases:
1. Should go to network if claimsBasedCaching is off and claims are present
2. Should go to cache (and match) if claimsBasedCaching is on and claims are present
3. Should go to network after going to cache if claimsBasedCaching is on and claims are present but different from cached claims
4. Should go to cache if claimsBasedCaching is off but no claims are present in request, returning a matching previously cached token even if it has requested claims